app/models/occupation_standard.rb

```diff
@@ -122,7 +132,7 @@ class OccupationStandard < ApplicationRecord
       indexes :rapids_code, type: :text, analyzer: :autocomplete, search_analyzer: :autocomplete_search
       indexes :state, type: :text, analyzer: :keyword
       indexes :state_id, type: :keyword
-      indexes :title, type: :text, analyzer: :rebuilt_english
+      indexes :title, type: :text, analyzer: :rebuilt_english, search_analyzer: :dynamic_synonym
```

Comment:
Since `rebuilt_english` isn't customized any longer, I think we should be able to remove the definition above, and just use `analyzer: :english` here.